### ISSUE TITLE
Wordpress detection fails

### ISSUE THREAD

**Issue by connorjclark** [johnmichel/Library-Detector-for-Chrome#203]
For wordpress.com, which I assume it built using wordpress... this library fails to detect it.

cc @housseindjirdeh  @westonruter

**Comment by westonruter** [comment]
Which URL on wordpress.com? On https://wordpress.com/?

**Comment by connorjclark** [comment]
Yeah, the home page... but only w/ mobile emulation enabled in devtools.

**Comment by connorjclark** [comment]
<img width="707" alt="image" src="https://user-images.githubusercontent.com/4071474/236055793-1f2d2a4d-36e7-4cfe-b752-45d741755a16.png">

Perhaps there is an A/B test? For me, on desktop `hasWPIncludes` is true (the other is false); but under mobile emulation `hasWPIncludes` is false. There is no `wlwmanifest` file for the mobile load (`<link rel="wlwmanifest" type="application/wlwmanifest+xml" href="//s1.wp.com/wp-includes/wlwmanifest.xml">`)

**Comment by westonruter** [comment]
Oh, I passed over those parts of the `test`:

https://github.com/johnmichel/Library-Detector-for-Chrome/blob/3135bb24072af715f45c8c54232c7a96d11b7b09/library/libraries.js#L1669-L1672

I'm somewhat surprised those conditions are part of the `test`. Someone can turn off the WordPress API, and thus `hasAPILinkElem` would be `false`. Similarly, a theme doesn't necessarily include any scripts or stylesheets that load from `wp-includes` (especially if an optimization plugin is processing them).

Also, it turns out that the `wlwmanifest.xml` link is [deprecated](https://core.trac.wordpress.org/ticket/41404) as of 6.3.0 (the next stable release) and will be removed from WordPress core. So it is expected that this wouldn't show up in the future:

```html
<link rel="wlwmanifest" type="application/wlwmanifest+xml" href="https://example.com/wp-includes/wlwmanifest.xml">
```

So the `link[href*="wp-includes"]` selector will _increasingly_ become unreliable for WordPress detection going forward. WordPress.com usually runs bleeding edge WordPress (pre-stable), which is perhaps why it is not showing up there now.

I suggest that the `hasAPILinkElem` and `hasWPIncludes` conditions be removed.

**Comment by westonruter** [comment]
It looks like originally when the test was introduced (2ab612aac3772b926908d7eb612d271ff8b7a386), it was checking for the `meta[name="generator"]` tag _or_ the page having a `script`/`link` that mentions `wp-includes`. This was then changed in a8a627667670e40016af987af6bc5bbfe27e5a69 to make it a both/and condition. See https://github.com/johnmichel/Library-Detector-for-Chrome/pull/131#pullrequestreview-210916940.

Pull request in question: #131
